src/js/utils/language.js

```diff
@@ -168,10 +171,16 @@ export function isLocalizationComplete(customLocalization) {
     });
 }
 
+// Used to ensure nb/nn language codes both return 'no'. 
+// TODO: Deprecate and replace with nn and nb
+function normalizeNorwegian(language) {
+    return /^n(b|n)$/.test(language) ? 'no' : language;
```

Comment:
When not using $1 in a `replace` or `match`, you don't need to capture what's in parenthesis `(b|n)`. To make the regex not capture what is in paranthesis you add `?:` at the start of the parenthesis like `/^n(?:b|n)$/`, but since these are single characters we can achieve the same and make the pattern shorter by using square brackets and dropping the OR pipe:
```suggestion
    return /^n[bn]$/.test(language) ? 'no' : language;
```